src/main/java/me/zombie_striker/qg/guns/Gun.java

```diff
@@ -160,7 +160,7 @@ public static boolean USE_THIS_INSTEAD_OF_INDEVIDUAL_SHOOT_METHODS(Gun g, Player
     }
 
     public static int getAmount(Player player) {
-        return getAmount(player.getInventory().getItemInHand());
+        return getAmount(player.getInventory().getItemInMainHand());
```

Comment:
This will throw errors in legacy versions. Please rollback that

Reply:
It exists on other classes, Ok with that?